blog/_src/posts/2018-08-16-if-you-give-a-mouse-a-macro.md

```diff
@@ -0,0 +1,445 @@
+    Title: If You Give a Mouse a Macro
```

Comment:
are you open to changing the title? how about something searchable, like "Implementing local define forms in Turnstile"

(or the pull request title: "define-like binding forms in Turnstile")

Reply:
Thanks for the feedback!

I am open to changing the title, but the descriptive titles I thought of seemed a bit dull.

Reply:
Yes, change the title please. How about "Type checking internal definition contexts with Turnstile"?

Reply:
Or maybe "Checking internal definition contexts via Types as Macros", since Turnstile is only superficially involved.

Reply:
I changed the title to "Defining Local Bindings in Turnstile Languages"

Reply:
This is nice, thanks!